src/index.html

```diff
@@ -7,8 +7,74 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header_style">
+      <div class="logo">
+        <a href="#home">
+          <img 
+          src="images/logo.png" 
+          alt="Logo" 
+          class="logo_image">
+        </a>
+      </div>
+      <nav>
+        <ul class="nav__list">
+          <li class="menu-item">
+            <a 
+            href="#" 
+            class="is-active menu_link"
+            >Apple</a>
```

Comment:
Fix the code style everywhere, please 
```suggestion
            <a href="#" class="is-active menu_link">
              Apple
            </a>
```

or 

```suggestion
            <a href="#" class="is-active menu_link">Apple</a>
```

Reply:
On this 

```
          <li class="menu-item">
            <a 
              href="#" 
              class="menu_link" 
              data-qa="hover"
            >
              Laptops &amp; computers
            </a>
          </li>
```
Had to brake it into several lines, othervise receiving errors

 error  Only 2 attributes per line are permitted  attr-new-line